lib/omnibus/packagers/msi.rb

```diff
@@ -178,6 +187,115 @@ def wix_candle_extension(extension)
     expose :wix_candle_extension
 
     #
+    # Set the name of the certificate store to use when signing packages.
+    #
+    # @example
+    #   cert_store_name "MY"
+    #
+    # @param [String] name
+    #   Name of the certificate store
+    #
+    # @return [String]
+    #   The name of the certificate store to be used
+    #
+    def cert_store_name(name = NULL)
```

Comment:
Can we please use a prefix of `singing_*` for these various signing config values. This would match the naming we use in other packagers and makes the purpose of these config values very obvious.